### PR TITLE
Shadows/Elevations

### PR DESCRIPTION
**Pull request by fraincs**
- Removed tables even/odd colors as it was not representing shadows well.
- Decreased font size one size in variables tables.
- Added two shadows to the main 2 shadows in order to have more flexibility.
- Color table now supports HSLA colors.
- Removed all semantic/alias classes.
- Added a concept of elevation / semantic shadows.
- TextButton padding is now on par with tags padding.
- Added a hover/focus chromatic test to buttons.
- Fixed a bug where a focused ghost button would lose its focus state while hovered
- Removed unnecessary margin in fluid buttons and tags containing icons and dots.
- Added chromatic test for invalid Select triggers.
- Removed browser default close button for SearchInput.

### CODE REVIEW

File: packages/react-components/src/select/tests/chromatic/createTestSuite.jsx [hunk 1]
```diff
@@ -266,13 +266,47 @@ export function createTestSuite(element, stories) {
...
+                    <Item key="saturn">Saturn</Item>
+                </Select>
                 <Select disabled placeholder="Select a planet" aria-label="Planets" element={element}>
                     <Item key="earth">Lorem Ipsum is simply dummy text of the printing and typesetting industry.</Item>
                     <Item key="mars">Mars</Item>
                     <Item key="saturn">Saturn</Item>
                 </Select>
             </Inline>
         )
+        .add("trigger states invalid", () =>
```
Comment (patricklafrance) [inline packages/react-components/src/select/tests/chromatic/createTestSuite.jsx:281]: these tests should either go in the "validation" test or in the "states" test

**Comment by patricklafrance** [comment]
There's a ")" missing in this table "Color Code" column:

image

**Comment by patricklafrance** [comment]
I think the tachyons classes in the docs are out of date:

image

**Comment by patricklafrance** [comment]
Seems like the "Variable" column still need an adjustment:

image

**Comment by patricklafrance** [comment]
I think the tachyons docs for the colors is also out of date, it using variables like `--o-ui-apricot-50` instead of `--o-ui-global-apricot-50`

image

**Comment by patricklafrance** [comment]
Remove Overcast?

image

**Comment by patricklafrance** [comment]
Remove "var(":

image

File: docs/materials/Shadows.stories.mdx [hunk 2]
```diff
@@ -17,20 +17,43 @@ All Tachyons shadows utilities have been replaced to match Orbit Design values.
...
+        [{ value: "shadow-100", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-100", className: "code f8 o-90" }, <div className="shadow-100 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-200", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-200", className: "code f8 o-90" }, <div className="shadow-200 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-300", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-300", className: "code f8 o-90" }, <div className="shadow-300 br2 bg-white h9 ma3"></div>],
+        [{ value: "shadow-400", className: "code f8 o-90" }, { value: "--o-ui-global-shadow-400", className: "code f8 o-90" }, <div className="shadow-400 br2 bg-white h9 ma3"></div>]
     ]}
     rowClassName="background-1"
 />
 
+CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows, the values below can be used [here](https://shadows.brumm.af/) in order to preview and play with the shadows.
```
Comment (patricklafrance) [inline docs/materials/Shadows.stories.mdx:31]: Would change 

"CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows, the values below can be used [here](https://shadows.brumm.af/) in order to preview and play with the shadows."

For:

CSS shadows are made using a system of 6 layers in order to provide silky smooth shadows. The shadows values can be previewed and altered on [shadows.brumm.af](https://shadows.brumm.af/).

**Comment by fraincs** [comment]
> I think the tachyons classes in the docs are out of date:
> 
> image

these are Tachyons classes and not 

> I think the tachyons docs for the colors is also out of date, it using variables like `--o-ui-apricot-50` instead of `--o-ui-global-apricot-50`
> 
> image

This bug has been flagged by me and is fixed in another branch, I could fix it here too actually

**Pull request merged**